src/Tizen.Multimedia.Metadata/MetadataExtractor/MetadataExtractorAttr.cs

```diff
@@ -34,7 +34,8 @@ internal enum MetadataExtractorAttr
         Album,
         AlbumArtist,
         Genre,
-        Author,
+        Author, // This will be deprecated. Please use `Composer` instead.
```

Comment:
This is for internal use, so you can remove it. Or, if you want to keep it as obsolete, 
```cs
[Obsolete("This will be deprecated. Please use `Composer` instead.")]
Author,
``` 
would be better.

Reply:
OK I'll remove it.